content/en/blog/_posts/2021-05-30-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog
+title: 'Updates to Ingress Nginx with network v1'
+date: 2021-05-30
+slug: update-with-ingress-nginx
+---
+
+**Authors:** James Strong, Richardo Katz 
+
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The network API is no
```

Comment:
There's a network API? Is that different from NetworkPolicy?

I know there's `networking.k8s.io/v1`; is this what we mean?

Reply:
Yes, networking.k8s.io/v1 . I can update the wording on that to add clarity.

Reply:
networking.k8s.io is actually an _API group_ not an API. Ingress, within that API group, is now GA.

Also: everything in in `networking.k8s.io/v1` is GA because `v1` represents, for Kubernetes a stable API version - no `beta` or `alpha`.

Reply:
My comment as well -- we should be specific. v1beta1 itself can't be deprecated because it's a just a (group, version). It's the Ingress API resource that is moving between versions.

Reply:
updated.